.github/workflows/ci.yml

```diff
@@ -12,6 +12,11 @@ env:
   # List of packages that will be checked with the minimum supported Rust version.
   # This should be limited to packages that are intended for publishing.
   RUST_MIN_VER_PKGS: "-p vello -p vello_encoding -p vello_shaders"
```

Comment:
I think these not having MSRV checked was an oversight (not that it matters when MSRV is MaxSRV, of course)

That is, we should add the sparse strips packages here.

Reply:
Done in PR #863.